### PR TITLE
Development of the header, main, and footer sections completed.

### PR DESCRIPTION
**Pull request by dannisonarias**
worked on the header, main, and footer sections and styled using CSS

### CODE REVIEW

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Comment by dannisonarias** [comment]
Sorry @kobiyoyo I fixed a few errors found by my coding partner last minute and I fixed them, please use a new live link

**Comment by kobiyoyo** [comment]
## Hi @dannisonarias      
#### Great Job so far, but there are a few more things required. 
### Navigation
- [ ] The navigation bar doesn't fill the page like the original site, let's fix that.
#### Original Version


#### Your Version


### Footer
- [ ] Footer should be similiar to orignal ,lets fix that. 
#### Original Version

#### Your Version

- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
### Make changes and submit another code review request.
*Happy coding*

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @dannisonarias      
#### Great Job so far, but there are a few more things required. 
### Navigation
- [ ] The navigation bar doesn't fill the page like the original site, let's fix that.
#### Original Version


#### Your Version


### Footer
- [ ] Footer should be similiar to orignal ,lets fix that. 
#### Original Version

#### Your Version

- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history (screenshot from Pathwright)
### Make changes and submit another code review request.
*Happy coding*

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,857 @@
...
+                    </div>
+                </div>
+
+            </div>
+            
+            
+        </article>
+        <article class="bottom-of-page container-fluid">
+            <div class="row">
+                <div class="col-12 bottom-of-page-title">
```
Comment (addod19) [inline index.html:435]: Let us add ```padding``` to ```<div class="bottom-of-page-title">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @dannisonarias, you have done a great job. 👍 Great working on the comments from previous reviewer. 
 For your project to be approved, we need to make some changes. Find comments below:
- Kindly edit the Pull Request so it has a title and a short summary of the project.
gh
- Let us fix the errors returned by [css validators](https://jigsaw.w3.org/css-validator)
rig
- Let us fix this 
ltfix 

Kindly make these changes and submit code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

File: index.html (hunk 1 above)
Comment (dannisonarias) [reply index.html:435]: I changed the property from "none" to "thin" to display that the property is working very well despite the CSS validator error.

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

A great effort in this project. Congratulations 🍺 . There are a few changes required, kindly fix it and submit code review:
- Missing contents on ``` mobile device```
wh

Good luck with your next submission ✈️ 
Happy coding 💻 
Daniel Larbi Addo

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello @dannisonarias ,  You have done a great job with this project. :clap:

**Pull request merged**